js/build/shortcode-ui.js

```diff
@@ -1273,7 +1272,7 @@ var EditShortcodeForm = wp.Backbone.View.extend({
 
 		} );
 
-		if ( 0 == this.model.get( 'attrs' ).length && ( ! innerContent || typeof innerContent == 'undefined' ) ) {
+		if ( 0 === this.model.get( 'attrs' ).length && ( ! innerContent || typeof innerContent == 'undefined' ) ) {
```

Comment:
The `typeof` comparison here should probably also use strict equality. Also, seems like general practice to check that the typeof an object isn't undefined before accessing the object.